lib/rubocop/rspec/language/node_pattern.rb

```diff
@@ -17,7 +17,11 @@ module NodePattern
 
         def_node_matcher :hook?, Hooks::ALL.block_pattern
 
-        def_node_matcher :let?, Helpers::ALL.block_pattern
+        def_node_matcher :let?, <<-PATTERN
+          {#{Helpers::ALL.block_pattern}
+           (send #{RSPEC} #{Helpers::ALL.node_pattern_union}
+                 _ (block_pass ...))}
```

Comment:
Instead of using `Helpers::ALL.send_pattern`, I thought it would be better to check for `block_pass` only.